developers/academy/zero_to_mvp/104_queries_2/20_hybrid.mdx

```diff
@@ -16,13 +16,13 @@ import imageUrl from '../../tmp_images/academy_placeholder.jpg';
 
 A hybrid search combines `bm25` searches that you just learned about with a vector search, producing rankings from a combination of the two results.
 
-This can produce helpful results when a vector search or a keyword search alone is not producing desired results. For example, it may be useful when a vector search alone is producing too many irrelevant results, and you want particular keywords to weight the results a certain way.
+This can produce helpful results when a vector search or a keyword search alone is not producing desired results. For example, it may be useful when a vector search alone is producing too many irrelevant results, and you want particular keywords to weigh the results a certain way.
```

Comment:
I think "weight" is correct here as it's describing like scaling one of vector or bm25 results more heavily than the other.